api/models.py

```diff
@@ -2,12 +2,12 @@
 
 db = SQLAlchemy()
 
-class Clothing(db.Model):
+class Clothing(db.Model):    
```

Comment:
[minor]
```suggestion
class Clothing(db.Model):
```

[dúvida] você já está usando alguma ferramenta de formatação de código?

Reply:
Ainda não! Não sei como implementar, mas pensei em fazer um Makefile. Funcionaria para o caso, não é?